cli/lib/cli/args.mjs

```diff
@@ -60,3 +60,7 @@ export async function* getURLs( opt ) {
 		}
 	}
 }
+
+export function logURLProgress( opt ) {
```

Comment:
Maybe this should be renamed to reflect its boolean nature, like `shouldLogProgress`. Otherwise the name sounds like _it_ is responsible for the logging.